### ISSUE TITLE
iosAfterPluginAdd Script also thinnes other frameworks

### ISSUE THREAD

**Issue by malua** [CleverTap/clevertap-cordova#45]
```
# This script loops through the frameworks embedded in the application and
# removes unused architectures.
find "$APP_PATH" -name \'*.framework\' -type d | while read -r FRAMEWORK
do
FRAMEWORK_EXECUTABLE_NAME=$(defaults read "$FRAMEWORK/Info.plist" CFBundleExecutable)
FRAMEWORK_EXECUTABLE_PATH="$FRAMEWORK/$FRAMEWORK_EXECUTABLE_NAME"
echo "Executable is $FRAMEWORK_EXECUTABLE_PATH"
EXTRACTED_ARCHS=()
for ARCH in $ARCHS
do
echo "Extracting $ARCH from $FRAMEWORK_EXECUTABLE_NAME"
lipo -extract "$ARCH" "$FRAMEWORK_EXECUTABLE_PATH" -o "$FRAMEWORK_EXECUTABLE_PATH-$ARCH"
EXTRACTED_ARCHS+=("$FRAMEWORK_EXECUTABLE_PATH-$ARCH")
done
echo "Merging extracted architectures: \${ARCHS}"
lipo -o "$FRAMEWORK_EXECUTABLE_PATH-merged" -create "\${EXTRACTED_ARCHS[@]}"
rm "\${EXTRACTED_ARCHS[@]}"
echo "Replacing original executable with thinned version"
rm "$FRAMEWORK_EXECUTABLE_PATH"
mv "$FRAMEWORK_EXECUTABLE_PATH-merged" "$FRAMEWORK_EXECUTABLE_PATH"
done`;
```

The script which is added to the ios build phases not only removes the unused architectures from Clevertap and SDWebImage, but also from e.g. the facebook sdk or branch sdk which leads to a broken build. To fix it you have to remove the wildcard * from the first line of the script.

For now as a workaround I have 2 scripts in the build phases.
one for clevertap:
`find "$APP_PATH" -name \'CleverTapSDK.framework\' -type d | while read -r FRAMEWORK`
and one for sdwebimage:
`find "$APP_PATH" -name \'SDWebImage.framework\' -type d | while read -r FRAMEWORK`

instead of the wildcard `*.framework`

thanks for you help!

**Comment by Aditi3** [comment]
Hi @malua 

Thank you for pointing this. We will add the change in our next release.

**Comment by darshanclevertap** [comment]
@malua this has been fixed in v2.1.9 which is live now

**Comment by Aditi3** [comment]
Hi @malua,

Considering all the scenarios, I think adding scripts at your end makes sense.

Additionally, you will be needing the wildcard(*) in the script to ensure that you don’t run any issues during the App Store deployment of the Application. 

Thank you!
